lib/generators/manageiq/plugin/templates/.github/workflows/ci.yaml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  schedule:
+  - cron: '0 0 * * *'
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version:
+<%= YAML.load_file(Rails.root.join(".github/workflows/ci.yaml")).dig("jobs", "ci", "strategy", "matrix", "ruby-version").to_a.map { |r| "        - #{r}" }.join("\n") %>
```

Comment:
Tried `.rjust()` but we don't know how long the version is going to be (might target a specific x.y.z version for example) and `8.times.map { " " }` was longer so... :)

Reply:
Oh interesting...I guess `indent` would be the method of choice, but this is fine :+1: